### PR TITLE
Coupla changes

### PR DESCRIPTION
**Pull request by cnnorris**
@mtphamine 

Opening PR in case you have questions

### CODE REVIEW

File: techtalk_slides.tex [hunk 1]
```diff
@@ -329,6 +344,7 @@ \section{Irregular Verbs}
 \end{frame}
 
 \begin{frame}{Best strategy}
+    %  TODO: these slides are too wordy, simplify
```
Comment (cnnorris) [inline techtalk_slides.tex:347]: Yeah @mtphamine I wasn't sure exactly what changes would accomplish this, but these two slides feel a little too "just read everything off the slide".

File: techtalk_slides.tex [hunk 2]
```diff
@@ -38,10 +37,11 @@ \section{Overview}
 	So what?
 	\pause
 
+    % TODO maybe: put in pictures instead of words?
```
Comment (mtphamine) [inline techtalk_slides.tex:40]: I think that the Zelda and Tay pictures afterwards are for towards this end?

File: techtalk_slides.tex [hunk 3]
```diff
@@ -197,6 +210,8 @@ \section{Overview}
 \end{frame}
 
 \begin{frame}{Outline of talk}
+    % TODO this also includes "overview" and "conclusion",
+    % do we need this slide at all?
```
Comment (mtphamine) [inline techtalk_slides.tex:214]: The outline command just generates all of it automatically. I think I know how to prevent it from generating the overview and conclusion in the outline, if that's better.
I think it's helpful to give a quick roadmap of a talk though, and this was an easier way to do it than making a handmade list
